components/Voluntary-Locking.jsx

```diff
@@ -86,9 +86,9 @@ function VoluntaryLocking({ network }) {
 		// can't be used to render a table (can't put a <table> in a <p>).
 		// So, we use the same component for Polkadot and Kusama and figure it out here.
 		const title = document.title;
-		if (title === "Governance · Polkadot Wiki") {
+		if (title === "Governance · Polkadot Wiki" || title === "Governance V2 · Polkadot Wiki") {
 			updateTable("polkadot")
-		} else if (title === "Governance · Guide") {
+		} else if (title === "Governance · Guide" || title === "Governance V2 · Guide") {
```

Comment:
This is required for the tables to render on the Gov2 doc.  Since the table was originally just present on the Gov1 doc, the page title name was used to differentiate which table to load (not the best mechanism) but this address it for now.